scss/_carousel.scss

```diff
@@ -302,8 +302,47 @@
 }
 
 /* rtl:end:ignore */
-// End mod
 
+.carousel-indicators-wrapper {
```

Comment:
So we keep this wording even if it is not only a wrapper for indicators but a wrapper for indicators + play/pause button?

Reply:
Unfortunalety I can't find an ideal solution.
`.carousel-controllers-wrapper` could be good but the prev/next buttons are already labelled as `controllers`. Maybe `.carousel-bottom-controllers-wrapper` but I don't like it...

Other ideas:
`.carousel-riding-controllers-wrapper`
`.carousel-bottom-bar-wrapper` or just `.carousel-bottom-bar`
anything else?

Reply:
@louismaximepiton, @hannahiss or @isabellechanclou , any ideas here to help us find a cute name for this new wrapper?

Reply:
`.carousel-meta-controllers` ? I can't think about anything else 🤔

Reply:
`.carousel-action-bar`?

Reply:
We collectively agreed to use `.carousel-action-bar`. I'll do the update in the PR.